cmd/juno/juno.go

```diff
@@ -237,8 +239,13 @@ func NewCmd(config *node.Config, run func(*cobra.Command, []string) error) *cobr
 		if v.IsSet(cnNameF) {
 			l1ChainID, ok := new(big.Int).SetString(v.GetString(cnL1ChainIDF), 0)
 			if !ok {
-				return fmt.Errorf("invalid L1 chain id %s", v.GetString(cnL1ChainIDF))
+				return fmt.Errorf("invalid %s id %s", cnL1ChainIDF, v.GetString(cnL1ChainIDF))
 			}
+			unverifRange := v.GetIntSlice(cnUnverifiableRangeF)
+			if len(unverifRange) != 2 || unverifRange[0] < 0 || unverifRange[1] < 0 {
+				return fmt.Errorf("invalid %s:%v, must be uint array of length 2 (e.g. `0,100`)", cnUnverifiableRangeF, unverifRange)
+			}
+
```

Comment:
Later on, let's move this (building custom `utils.Network` from config) to it's own function.